src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -84980,5 +84980,30 @@ static void MN2()
                 //         node.ToString(); // 8
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "node").WithLocation(118, 9));
         }
+
+        [Fact, WorkItem(32446, "https://github.com/dotnet/roslyn/issues/32446")]
+        public void RefValueOfNullableType()
+        {
+            var source =
+@"
+#nullable enable
```

Comment:
`#nullable enable` is unnecessary, since you already have `options: WithNonNullTypesTrue()`